realtime.js

```diff
@@ -145,3 +145,20 @@ exports.waittimes = function(times) {
         times: times
     });
 }
+
+// Takes in a list of tas to notify about their help time
+exports.notifytime = function(notif_tas) {
+    exports.seq = exports.seq + 1;
```

Comment:
The point of `seq` is to ensure that the incremental updates are all applied correctly, and to force the client to reload if one of them is missed (meaning we have inconsistent queue data). We don't really need to use `seq` for this feature since it isn't critical if one of the updates is missed.